client/layout/index.jsx

```diff
@@ -143,8 +144,10 @@ Layout = React.createClass( {
 export default connect(
 	( state ) => {
 		const { isLoading, section, hasSidebar, chunkName } = state.ui;
+		const isSupportUser = state.support.isSupportUser;
```

Comment:
Looking at this here, it should probably be a selector: `isSupportUser( state )`